naga/src/back/spv/block.rs

```diff
@@ -59,6 +59,30 @@ enum BlockExit {
     },
 }
 
+/// What code generation did with a provided [`BlockExit`] value.
+///
+/// A function that accepts a [`BlockExit`] argument should return a value of
```

Comment:
nitpick(non-blocking): We only use this for a single function, so it's sorta weird to say "a function" like it might be typical to use in multiple places. Not really a concern, tho.